.travis.yml

```diff
@@ -4,6 +4,8 @@ language: python
 branches:
   only:
     - master
+    - v0.3.0-dev
```

Comment:
What is this referring to?

Reply:
Just another branch that is in the repo.

Reply:
Sure. Why this branch specifically?

Reply:
It was the only other dev branch. I can remove it.